dirsrvtests/tests/suites/clu/dsconf_test.py

```diff
@@ -0,0 +1,87 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+"""Test dsconf CLI with LDAPS"""
+
+import logging
+import pytest
+import subprocess
+import os
+from lib389._constants import DEFAULT_SUFFIX, DN_DM
+from lib389.topologies import topology_st
+
+pytestmark = pytest.mark.tier1
+
+log = logging.getLogger(__name__)
+
+@pytest.fixture(scope="function")
+def enable_config(request, topology_st, config_type):
+    if config_type == 'ldapfile':
+        with open('/etc/openldap/ldap.conf', 'a') as f:
+            f.write("TLS_CACERT /etc/dirsrv/slapd-standalone1/ca.crt\n")
+            f.close()
+
+    else:
+        data = ['[localhost]\n', 'tls_cacertdir = /etc/dirsrv/slapd-standalone1\n',
+                f'uri = {topology_st.standalone.get_ldaps_uri()}\n']
+        fd = open('/root/.dsrc', 'w')
+        for line in data:
+            fd.write(line)
+        fd.close()
+
+    def fin():
```

Comment:
Not sure if this is the right way of handling this, need suggestions, Thanks!

Reply:
yeah, this isn't a good way to do it. Remember, the test could be run as someone who is not-root (ie dirsrv user), and each time this test is run it's going to keep adding this. Imagine the test fails and fin isn't run, then it will remain for example? 

A better way is that the libldap tools read from the environment with each variable in ldap.conf prefixed with LDAP. 

For example  `LDAPTLS_CACERT=/path/to/cert dsconf ....`

that should allow an alternate way to set this value that is non destructive and should have the same effect. 

Hope that helps!

Reply:
@Firstyear Hey thanks for your review, this bug is related to ```ldap.conf``` specifically so I think it's better to use ```LDAPCONF``` in this way we can be sure that settings from ```ldap.conf``` are respected correctly, let me know what are you thoughts on this. :)